js/stringProcessing/countSyllables.js

```diff
@@ -63,15 +65,29 @@ var countSyllablesInExclusions = function( word ) {
 };
 
 /**
+ * Counts syllables in partial exclusions. If these are found, the number of syllables are found, and the modified word, as to not do
```

Comment:
This description seems a bit unclear to me. The second sentence doesn't seem to make a whole lot of sense (possibly due to it's length).